control-plane/api-gateway/controllers/gateway_controller.go

```diff
@@ -13,6 +14,7 @@ import (
 	"github.com/hashicorp/consul-k8s/control-plane/connect-inject/constants"
 
 	"github.com/go-logr/logr"
+	"github.com/pkg/errors"
```

Comment:
did you mean to add this?

Reply:
Yes, I was using the errors.Wrap() function in that package. But that'll probably go away after I figure out how to bubble up the errors like I want

Reply:
gotcha, would the error wrapping in the std lib solve that?

Reply:
Probably, but no longer using the wrap function.